pkg/cmd/pipeline/logs.go

```diff
@@ -129,11 +137,16 @@ func (opts *logOptions) init(args []string) error {
 }
 
 func (opts *logOptions) getAllInputs() error {
-	ps, err := allPipelines(opts.params)
+	ps, err := opts.allPipelines()
```

Comment:
This is not consistent with `verify`. Why this change ?

Reply:
`verify()` checks whether the input which is pipeline given by user exist or not.
`getAllInputs()` takes no input from the user and given options to select from.

`tkn pipeline logs foo` showed message no pipelinerun found for the pipeline foo even if the pipeline itself is not there, `verify()` checks if the pipeline exists given by user.

Reply:
my point is on the `go` level. somewhere we do `opts.allPipelines()` and elsewhere we do `verify(opts.pipelineName, opts.param)`. Why not keep `allPipelines(opts.params)` then ?

Reply:
Make sense, I will make it consistent and update this.